script.js

```diff
@@ -152,12 +155,18 @@ async function OptimizeParams(userInputs, tvParameterIndex, optimizationResults)
         avgerageBarsInTrades: 0
     });
     setTimeout(() => {
-        // Hover on Input Arrows  
+        // Hover on Input Arrows
         tvInputs[tvParameterIndex].dispatchEvent(new MouseEvent('mouseover', { 'bubbles': true }));
     }, 250);
     setTimeout(() => {
         // Click on Upper Input Arrow
         document.querySelectorAll("button[class*=controlIncrease]")[tvParameterIndex].click()
+        // If Deep Backtesting is enabled, click on Generate Report Button
```

Comment:
I think I got the idea, but not sure if we should put it within main optimization logic because this may increase complexity for basic flow and may possible break it in future.